Python/pywarpx/picmi.py

```diff
@@ -690,13 +690,13 @@ def initialize_inputs(self):
         pywarpx.algo.charge_deposition = self.charge_deposition_algo
         pywarpx.algo.field_gathering = self.field_gathering_algo
         pywarpx.algo.particle_pusher = self.particle_pusher_algo
+        pywarpx.algo.load_balance_intervals = self.load_balance_intervals
+        pywarpx.algo.load_balance_with_sfc = self.load_balance_with_sfc
 
```

Comment:
@dpgrote @mrowan137 we need to add the new `load_balance_knapsack_factor` option here, too :)

Reply:
@dpgrote @ax3l I've added load balance parameters in `picmi.py`, can I get a double check?

Reply:
This looks good!